cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -25,25 +25,32 @@ import (
 )
 
 var (
-	service  = flag.String("service", "", "service to generate config for")
-	outDir   = flag.String("out", os.Getenv("RUNTIME_DIRECTORY"), "directory to write configuration files to")
-	input    = flag.String("in", "/etc/google-cloud-ops-agent/config.yaml", "path to the user specified agent config")
-	logsDir  = flag.String("logs", "/var/log/google-cloud-ops-agent", "path to store agent logs")
-	stateDir = flag.String("state", "/var/lib/google-cloud-ops-agent", "path to store agent state like buffers")
+	service      = flag.String("service", "", "service to generate config for")
+	outDir       = flag.String("out", os.Getenv("RUNTIME_DIRECTORY"), "directory to write configuration files to")
+	input        = flag.String("in", "/etc/google-cloud-ops-agent/config.yaml", "path to the user specified agent config")
+	logsDir      = flag.String("logs", "/var/log/google-cloud-ops-agent", "path to store agent logs")
+	stateDir     = flag.String("state", "/var/lib/google-cloud-ops-agent", "path to store agent state like buffers")
+	healthChecks = flag.Bool("healthchecks", false, "run health checks and exit")
 )
 
-func runStartupChecks(service string) {
-	if service == "" {
-		gceHealthChecks := healthchecks.HealthCheckRegistryFactory()
-		logger, closer := healthchecks.CreateHealthChecksLogger(*logsDir)
-		defer closer()
+func runHealthChecks(registry healthchecks.HealthCheckRegistry) {
```

Comment:
Also, we should also add this in the Windows service too :
https://github.com/GoogleCloudPlatform/ops-agent/blob/master/cmd/ops_agent_windows/run_windows.go#L136-L151

If this feels to big of a change, maybe we can only add the new `Network Checks` here and defer the `--healthchecks` fucntion to a followup `PR`.